core/src/androidMain/kotlin/Scanner.kt

```diff
@@ -26,7 +26,7 @@ public class AndroidScanner internal constructor(
     logging: Logging,
 ) : Scanner {
 
-    private val logger = Logger(logging, tag = "Kable/Scanner")
+    private val logger = Logger(logging, tag = "Kable/Scanner", identifier = "")
```

Comment:
Seems like `identifier = null` might be more appropriate?

Reply:
Good call.